src/index.css

```diff
@@ -2,14 +2,12 @@
 @tailwind components;
 @tailwind utilities;
 
-/* We can use this selector to target active NavLinks */
-a.active {
-	@apply text-secondary;
-}
+/*
 
```

Comment:
Remember to use base, components, utilities layers.
@layer base -> All of your HTML Element selectors
@layer components -> All of your component custom classes
@layer utilities -> For utility classes that you most likely only use once